### PR TITLE
Cookiecut (first time)

### PR DESCRIPTION
**Pull request by bobleesj**
Cookiecut for the first time

Screenshot 2024-09-30 at 3 41 11 PM

Docs running nicely.

### CODE REVIEW

**Comment by bobleesj** [comment]
Ready for review @sbillinge 

Half of the changes are due to doc files generated automatically.

File: README.rst [hunk 1]
```diff
@@ -35,7 +64,14 @@ co-refinements with other techniques.
 For more information about the diffpy.srfit library, see the users manual at
 http://diffpy.github.io/diffpy.srfit.
 
-REQUIREMENTS
+Citation
+--------
+
+If you use diffpy.srfit in a scientific publication, we would like you to cite this package as
+
+        diffpy.srfit Package, https://github.com/diffpy/diffpy.srfit
```
Comment (sbillinge) [inline README.rst:72]: please have them cite the diffpy-cmi complex modeling paper.   Please have them cite this same paper for:
srfit
srreal
structure
utils
...am I forgetting any? not pdfmorph/snmf/fourigui which are separate packages.

File: README.rst [hunk 2]
```diff
@@ -79,68 +115,79 @@ Linux some of the required software can be installed using ::
...
-------------------------------------------------------------------------
+`Diffpy user group <https://groups.google.com/g/diffpy-users>`_ is the discussion forum for general questions and discussions about the use of diffpy.srfit. Please join the diffpy.srfit users community by joining the Google group. The diffpy.srfit project welcomes your expertise and enthusiasm!
+
+If you see a bug or want to request a feature, please `report it as an issue <https://github.com/diffpy/diffpy.srfit/issues>`_ and/or `submit a fix as a PR <https://github.com/diffpy/diffpy.srfit/pulls>`_. You can also post it to the `Diffpy user group <https://groups.google.com/g/diffpy-users>`_.
 
 diffpy.srfit is an open-source software developed as a part of the DiffPy-CMI
-complex modeling initiative at the Brookhaven National Laboratory.  The
-diffpy.srfit sources are hosted at
-https://github.com/diffpy/diffpy.srfit.
+complex modeling initiative at the Brookhaven National Laboratory.
```
Comment (sbillinge) [inline README.rst:159]: please change this to Columbia University.

**Review by sbillinge — COMMENTED** [review]
just a couple of inline comments

File: README.rst [hunk 3]
```diff
@@ -156,7 +163,7 @@ Support and Contribute
 If you see a bug or want to request a feature, please `report it as an issue <https://github.com/diffpy/diffpy.srfit/issues>`_ and/or `submit a fix as a PR <https://github.com/diffpy/diffpy.srfit/pulls>`_. You can also post it to the `Diffpy user group <https://groups.google.com/g/diffpy-users>`_.
 
 diffpy.srfit is an open-source software developed as a part of the DiffPy-CMI
-complex modeling initiative at the Brookhaven National Laboratory.
+complex modeling initiative at Columbia University.
```
Comment (bobleesj) [inline README.rst:166]: Change to Columbia

File: README.rst [hunk 4]
```diff
@@ -67,9 +67,16 @@ http://diffpy.github.io/diffpy.srfit.
 Citation
 --------
 
-If you use diffpy.srfit in a scientific publication, we would like you to cite this package as
+If you use this program for a scientific research that leads
```
Comment (bobleesj) [inline README.rst:70]: Added citation

- made PRs to diffpy.srreal and diffpy.utils, adding citation

**Comment by bobleesj** [comment]
@sbillinge ready for review

**Pull request merged**